wazuh/wazuh-dashboard-plugins
# Fix cannot perform a react state update on an unmounted component

### Description

Fix cannot perform a React state update on an unmounted component in the framework tab of the MITRE ATT&CK module, in the inventory data and stats tabs of the endpoint summary module. 
### Issues Resolved

#6693 

### Evidence

<img width="1792" alt="image" src="https://github.com/wazuh/wazuh-dashboard-plugins/assets/99441266/0e3e9dba-5991-457d-9e9b-60ba2eca7a9d">

<img width="1792" alt="image" src="https://github.com/wazuh/wazuh-dashboard-plugins/assets/99441266/2b0b681f-aeca-4bb7-ab18-2bbfd1c5301f">

<img width="1792" alt="image" src="https://github.com/wazuh/wazuh-dashboard-plugins/assets/99441266/faf42571-f1ca-4cf2-9a8c-0889d90712c5">

<img width="575" alt="image" src="https://github.com/wazuh/wazuh-dashboard-plugins/assets/99441266/9199f179-46a4-4789-a81c-5c26e25a625f">


### Test

- Go to the Framework tab of the MITRE ATT&CK module and verify that there is no warning in the console with the message Can't perform a React state update on an unmounted component.
- Go to the Endpoint summary module, select an agent then select inventory data and check that there is no warning in the console with the message Can't perform a React state update on an unmounted component.
- Go to the Endpoint summary module, select an agent then select stats and check that there is no warning in the console with the message Can't perform a React state update on an unmounted component.

### Check List
